telebot/__init__.py

```diff
@@ -613,6 +632,134 @@ def kick_chat_member(self, chat_id, user_id):
     def unban_chat_member(self, chat_id, user_id):
         return apihelper.unban_chat_member(self.token, chat_id, user_id)
 
+    def restrict_chat_member(self, chat_id, user_id):
```

Comment:
Missing some not require params. https://core.telegram.org/bots/api#restrictchatmember